daemon/policy.go

```diff
@@ -225,21 +278,29 @@ func (d *Daemon) PolicyAdd(rules policyAPI.Rules, opts *AddOptions) (newRev uint
 		metrics.PolicyImportErrors.Inc()
 		logger.WithError(err).WithField("prefixes", prefixes).Warn(
 			"Failed to allocate identities for CIDRs during policy add")
-		return 0, err
+		resChan <- &PolicyAddResult{
+			newRev: 0,
+			err:    err,
+		}
+		return
 	}
 
+	// No errors past this point!
```

Comment:
Can you elaborate on this comment? Do you not expect any errors past this point, or any error past this point will crash the daemon?

Reply:
Went ahead and removed it since it doesn't add value. 

Basically, there should be no more errors related to policy being added past this point because we've done all operations that could fail.